teleport_tube.lua

```diff
@@ -242,6 +243,14 @@ local function can_go(pos, node, velocity, stack)
 		return {}
 	end
 	local target = receivers[math.random(1, #receivers)]
+	local src_owner = src_meta:get_string("owner")
```

Comment:
not sure if this is performance-critical, but can this be moved inside the `if` clause below? thanks :+1:

Reply:
admittedly sloppy on my part, i've rescoped those 3 calculations.